climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1108 @@
+
+        Exposures.__init__(
+            self,
+            data=litpop_gdf,
+            crs=litpop_gdf.crs,
+            ref_year=reference_year,
+            tag=tag,
+            value_unit=value_unit
+        )
+
+        try:
+            rows, cols, ras_trans = u_coord.pts_to_raster_meta(
+                (self.gdf.longitude.min(), self.gdf.latitude.min(),
+                 self.gdf.longitude.max(), self.gdf.latitude.max()),
+                u_coord.get_resolution(self.gdf.longitude, self.gdf.latitude))
+            self.meta = {
+                'width': cols,
+                'height': rows,
+                'crs': self.crs,
+                'transform': ras_trans,
+            }
+        except ValueError:
+            LOGGER.warning('Could not write attribute meta, because exposure'
+                           ' has only 1 data point')
+            self.meta = {}
+
+    def set_lit(self, countries=None, shape=None, res_arcsec=15, exponent=1,
+                reference_year=2016, data_dir=None):
+        """
+        initiate Exposure with pure nightlight data
+
+        Parameters
+        ----------
+        countries : list or str (optional)
+            list containing country identifiers (name or iso3)
+        shape : Shape, Polygon or MultiPolygon, optional
+            shape to crop results to, overrules countries
+        res_arcsec : int, optional
+            Resolution in arc seconds. The default is 15.
+        exponent : int or float >= 0, optional
+            exponent m in Lit^m. The default is 1.
+        reference_year : int, optional
+            Target year, closest BlackMarble year is returned.
+            The default is 2016.
+        data_dir : Path, optional
+            data directory. The default is None.
+
+        Raises
+        ------
+        ValueError
+            Either countries or shape is required.
+        """
+
+        if countries is None and shape is None:
+            raise ValueError("Either countries or shape required. Aborting.")
+        if shape is None:
+            self.set_countries(countries, res_km=None, res_arcsec=res_arcsec,
+                               exponents=(exponent, 0), fin_mode='none', 
+                               reference_year=reference_year, gpw_version=None,
+                               data_dir=data_dir, resample_first=True)
+        elif countries is not None:
+            LOGGER.info("Using shape provided instead of countries' shapes. "
+                        "To use countries' shapes, set shape=None")
+        if shape is not None:
+            self.set_custom_shape(shape, res_arcsec=res_arcsec, exponents=(exponent,0),
+                                  fin_mode='none', in_countries=countries,
+                                  reference_year=reference_year,
+                                  gpw_version=None, data_dir=data_dir,
+                                  resample_first=True)
```

Comment:
```suggestion
        if shape is not None:
            if countries is not None:
                 LOGGER.info("Using shape provided instead of countries' shapes. "
                        "To use countries' shapes, set shape=None")
            self.set_custom_shape(shape, res_arcsec=res_arcsec, exponents=(exponent,0),
                                  fin_mode='none', in_countries=countries,
                                  reference_year=reference_year,
                                  gpw_version=None, data_dir=data_dir,
                                  resample_first=True)
```